2020-07-23 11:24:34.207 [http-nio-8085-exec-2] WARN  com.zaxxer.hikari.util.DriverDataSource - Registered driver with driverClassName=com.mysql.jdbc.Driver was not found, trying direct instantiation.
2020-07-23 12:18:22.326 [lettuce-nioEventLoop-6-2] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: Reconnection attempt exceeded timeout of 60 SECONDS 
2020-07-23 12:18:32.392 [lettuce-nioEventLoop-6-3] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:18:36.800 [HikariPool-1 housekeeper] WARN  com.zaxxer.hikari.pool.HikariPool - HikariPool-1 - Thread starvation or clock leap detected (housekeeper delta=2m31s887ms37Âµs585ns).
2020-07-23 12:18:42.655 [lettuce-nioEventLoop-6-4] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:18:52.676 [lettuce-nioEventLoop-6-1] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:19:02.777 [lettuce-nioEventLoop-6-2] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:19:12.877 [lettuce-nioEventLoop-6-3] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:19:22.975 [lettuce-nioEventLoop-6-4] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:19:33.177 [lettuce-nioEventLoop-6-1] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:19:43.477 [lettuce-nioEventLoop-6-2] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:19:54.078 [lettuce-nioEventLoop-6-3] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:20:05.177 [lettuce-nioEventLoop-6-4] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:20:17.277 [lettuce-nioEventLoop-6-1] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:20:31.477 [lettuce-nioEventLoop-6-2] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:20:49.676 [lettuce-nioEventLoop-6-3] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:21:16.078 [lettuce-nioEventLoop-6-4] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:21:56.177 [lettuce-nioEventLoop-6-1] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:22:36.277 [lettuce-nioEventLoop-6-2] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:23:16.375 [lettuce-nioEventLoop-6-3] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:23:56.486 [lettuce-nioEventLoop-6-4] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:24:36.611 [lettuce-nioEventLoop-6-1] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:25:16.737 [lettuce-nioEventLoop-6-2] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:25:56.865 [lettuce-nioEventLoop-6-3] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:26:36.990 [lettuce-nioEventLoop-6-4] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:27:17.115 [lettuce-nioEventLoop-6-1] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:27:57.240 [lettuce-nioEventLoop-6-2] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:28:37.364 [lettuce-nioEventLoop-6-3] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:29:17.468 [lettuce-nioEventLoop-6-4] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:29:57.568 [lettuce-nioEventLoop-6-1] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:30:37.668 [lettuce-nioEventLoop-6-2] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:31:17.768 [lettuce-nioEventLoop-6-3] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:31:57.868 [lettuce-nioEventLoop-6-4] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:32:37.968 [lettuce-nioEventLoop-6-1] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:33:18.068 [lettuce-nioEventLoop-6-2] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:33:58.169 [lettuce-nioEventLoop-6-3] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:34:38.268 [lettuce-nioEventLoop-6-4] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:35:18.367 [lettuce-nioEventLoop-6-1] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:35:58.469 [lettuce-nioEventLoop-6-2] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:36:38.570 [lettuce-nioEventLoop-6-3] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:37:18.669 [lettuce-nioEventLoop-6-4] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:37:58.771 [lettuce-nioEventLoop-6-1] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:38:38.873 [lettuce-nioEventLoop-6-2] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:39:18.971 [lettuce-nioEventLoop-6-3] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:39:59.073 [lettuce-nioEventLoop-6-4] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:40:39.171 [lettuce-nioEventLoop-6-1] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:41:19.274 [lettuce-nioEventLoop-6-2] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:41:59.370 [lettuce-nioEventLoop-6-3] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:42:39.472 [lettuce-nioEventLoop-6-4] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:43:19.578 [lettuce-nioEventLoop-6-1] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:43:59.671 [lettuce-nioEventLoop-6-2] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:44:39.773 [lettuce-nioEventLoop-6-3] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:45:19.871 [lettuce-nioEventLoop-6-4] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:45:59.974 [lettuce-nioEventLoop-6-1] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:46:40.075 [lettuce-nioEventLoop-6-2] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:47:20.178 [lettuce-nioEventLoop-6-3] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:48:00.281 [lettuce-nioEventLoop-6-4] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:48:40.384 [lettuce-nioEventLoop-6-1] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:49:20.488 [lettuce-nioEventLoop-6-2] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:50:00.591 [lettuce-nioEventLoop-6-3] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:50:40.694 [lettuce-nioEventLoop-6-4] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:51:20.798 [lettuce-nioEventLoop-6-1] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:52:00.898 [lettuce-nioEventLoop-6-2] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:52:40.998 [lettuce-nioEventLoop-6-3] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:53:21.098 [lettuce-nioEventLoop-6-4] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:54:01.198 [lettuce-nioEventLoop-6-1] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:54:41.298 [lettuce-nioEventLoop-6-2] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:55:21.399 [lettuce-nioEventLoop-6-3] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:56:01.498 [lettuce-nioEventLoop-6-4] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:56:41.597 [lettuce-nioEventLoop-6-1] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:57:21.698 [lettuce-nioEventLoop-6-2] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:58:01.799 [lettuce-nioEventLoop-6-3] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:58:41.896 [lettuce-nioEventLoop-6-4] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 12:59:21.999 [lettuce-nioEventLoop-6-1] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 13:00:02.140 [lettuce-nioEventLoop-6-2] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 13:00:42.196 [lettuce-nioEventLoop-6-3] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 13:01:22.296 [lettuce-nioEventLoop-6-4] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 13:02:02.393 [lettuce-nioEventLoop-6-1] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 13:02:42.491 [lettuce-nioEventLoop-6-2] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 13:03:22.592 [lettuce-nioEventLoop-6-3] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 13:04:02.691 [lettuce-nioEventLoop-6-4] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 13:04:42.800 [lettuce-nioEventLoop-6-1] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 13:05:22.892 [lettuce-nioEventLoop-6-2] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 13:06:02.991 [lettuce-nioEventLoop-6-3] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 13:06:43.092 [lettuce-nioEventLoop-6-4] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 13:07:23.194 [lettuce-nioEventLoop-6-1] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 13:08:03.293 [lettuce-nioEventLoop-6-2] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 13:08:43.392 [lettuce-nioEventLoop-6-3] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 13:09:23.492 [lettuce-nioEventLoop-6-4] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 13:10:03.592 [lettuce-nioEventLoop-6-1] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 13:10:43.693 [lettuce-nioEventLoop-6-2] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 13:11:23.791 [lettuce-nioEventLoop-6-3] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 13:12:03.895 [lettuce-nioEventLoop-6-4] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 13:12:44.031 [lettuce-nioEventLoop-6-1] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 13:13:24.093 [lettuce-nioEventLoop-6-2] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 13:14:04.194 [lettuce-nioEventLoop-6-3] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 13:14:44.308 [lettuce-nioEventLoop-6-4] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 13:15:24.391 [lettuce-nioEventLoop-6-1] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 13:16:04.493 [lettuce-nioEventLoop-6-2] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
2020-07-23 13:16:44.612 [lettuce-nioEventLoop-6-3] WARN  io.lettuce.core.protocol.ConnectionWatchdog - Cannot reconnect to [192.168.30.10:6379]: connection timed out: /192.168.30.10:6379
